core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4204,6 +4204,14 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.WORKER)
           .build();
+  public static final PropertyKey WORKER_NIO_DIRECT_BUFFER_POOL_ENABLED =
+      booleanBuilder(Name.WORKER_NIO_DIRECT_BUFFER_POOL_ENABLED)
+          .setDefaultValue(true)
+          .setDescription("Whether it is using pooled direct buffer or unpooled wrapped buffer"
+              + " when creating a buffer for remote read")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.WORKER)
```

Comment:
I think this is a temporary and experimental switch. When we finalize on which is the best option for handling buffers, this switch should be removed.

```suggestion
          .setScope(Scope.WORKER)
          .setHidden(true)
```